elements/gotofail-drawer.html

```diff
@@ -72,6 +74,10 @@
                 background-color: transparent;
             }
 
+            .ripple-style {
```

Comment:
You can change this to `paper-ripple` and remove the added classes on the `paper-ripple` elements.